src/guiFormSpecMenu.cpp

```diff
@@ -2432,18 +2444,20 @@ void GUIFormSpecMenu::drawMenu()
 		Phase 0: Item slot rectangles
 		Phase 1: Item images; prepare tooltip
 	*/
+	bool item_hovered = false;
 	int start_phase=0;
 	for(int phase=start_phase; phase<=1; phase++)
-	for(u32 i=0; i<m_inventorylists.size(); i++)
+	for (u32 i=0; i<m_inventorylists.size(); i++) 
 	{
-		drawList(m_inventorylists[i], phase);
+		drawList(m_inventorylists[i], phase, item_hovered);
 	}
-
-	/*
-		Call base class
-	*/
-	gui::IGUIElement::draw();
-
+	if (!item_hovered) {
+		ItemStack item;
+		drawItemStack(driver, m_font, item,
```

Comment:
you can pass directly a ItemStack() instead of allocating a variable and passing it